ladybug/src/main/java/org/frankframework/ibistesttool/DeploymentSpecificsBeanPostProcessor.java

```diff
@@ -92,6 +93,14 @@ public Object postProcessBeforeInitialization(Object bean, String beanName) thro
 				}
 			}
 		}
+		if (bean instanceof SpringLiquibase) {
+			boolean active = APP_CONSTANTS.getBoolean("ladybug.jdbc.migrator.active",
+					APP_CONSTANTS.getBoolean("jdbc.migrator.active", false));
+			if (!active) {
```

Comment:
Waarom niet gewoon altijd de waarde van `active` in de bean zetten?

Reply:
Als private field (bovenaan in de classe)